Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/SettingsFile.java

```diff
@@ -277,7 +277,10 @@ private SettingsFile()
 				else if ((current != null) && line.contains("="))
```

Comment:
You should be able to remove the `line.contains("=")` check now, since it's covered by your new `splitLine.length != 2` check.